UITests/Helper/TransferFunds/TransferFundsConfirmation.swift

```diff
@@ -0,0 +1,67 @@
+import Common
+import XCTest
+
+class TransferFundsConfirmation {
+    var app: XCUIApplication
+
+    // Destination Section
+    var addSelectDestinationLabel: XCUIElement
```

Comment:
The naming of this is confusing, there shouldn't be any selection or add in this page.  You can rename it to transferDestinationLabel and transferDestinationDetail.